tests/braille-specs/kk.yaml

```diff
@@ -0,0 +1,39 @@
+# Tests for Kazakh literary braille
+#
+# Copyright © 2018 by Sergiy Moskalets <www.trosti.com.ua>
+# Copyright © 2021 by Andrey Yakuboy <andrewia2002@yandex.ru>
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+# ----------------------------------------------------------------------------------------------
+
+display: unicode-without-blank.dis,ru-unicode.dis
+table:
+  locale: kk
+  type: literary
+  __assert-match: kk.utb
+  contraction: no
+  dots: 6
+tests:
+  - - Барлық адамдар тумасынан азат және қадір-қасиеті мен құқықтары тең болып дүниеге келеді. Адамдарға ақыл-парасат, ар-ождан берілген, сондықтан олар бір-бірімен туыстық, бауырмалдық қарым-қатынас жасаулары тиіс.
+    - ⠃⠁⠗⠇⠮⠹ ⠁⠙⠁⠍⠙⠁⠗ ⠞⠥⠍⠁⠎⠮⠝⠁⠝ ⠁⠵⠁⠞ ⠚⠜⠝⠑ ⠹⠁⠙⠽⠗⠤⠹⠁⠎⠊⠑⠞⠽ ⠍⠑⠝ ⠹⠬⠹⠮⠹⠞⠁⠗⠮ ⠞⠑⠩ ⠃⠕⠇⠮⠏ ⠙⠌⠝⠊⠑⠛⠑ ⠅⠑⠇⠑⠙⠽⠲ ⠁⠙⠁⠍⠙⠁⠗⠻⠁ ⠁⠹⠮⠇⠤⠏⠁⠗⠁⠎⠁⠞⠂⠁⠗⠤⠕⠚⠙⠁⠝ ⠃⠑⠗⠽⠇⠛⠑⠝⠂⠎⠕⠝⠙⠮⠹⠞⠁⠝ ⠕⠇⠁⠗ ⠃⠽⠗⠤⠃⠽⠗⠽⠍⠑⠝ ⠞⠥⠮⠎⠞⠮⠹⠂⠃⠁⠥⠮⠗⠍⠁⠇⠙⠮⠹ ⠹⠁⠗⠮⠍⠤⠹⠁⠞⠮⠝⠁⠎ ⠚⠁⠎⠁⠥⠇⠁⠗⠮ ⠞⠊⠽⠎⠲
+  - - 123,5+46=169,5
+    - ⠼⠁⠃⠉⠂⠑ ⠖⠼⠙⠋ ⠶⠼⠁⠋⠊⠂⠑
+  - - 1, 3, 5, 7, 11, 13, 17, 19, 23
+    - ⠼⠁⠠⠂⠼⠉⠠⠂⠼⠑⠠⠂⠼⠛⠠⠂⠼⠁⠁⠠⠂⠼⠁⠉⠠⠂⠼⠁⠛⠠⠂⠼⠁⠊⠠⠂⠼⠃⠉
+  - - (1 + 2 + 3) + 4 = 10
+    - ⠣⠼⠁ ⠖⠼⠃ ⠖⠼⠉⠜ ⠖⠼⠙ ⠶⠼⠁⠚
+  - - Тынық мұхиттың оңтүстік тарабындағы Самоа архипелагінің (шоқ аралының) батыс аралдарында орналасқан.
+    - ⠞⠮⠝⠮⠹ ⠍⠬⠓⠊⠞⠞⠮⠩ ⠕⠩⠞⠌⠎⠞⠽⠅ ⠞⠁⠗⠁⠃⠮⠝⠙⠁⠻⠮ ⠎⠁⠍⠕⠁ ⠁⠗⠓⠊⠏⠑⠇⠁⠛⠽⠝⠽⠩ ⠶⠱⠕⠹ ⠁⠗⠁⠇⠮⠝⠮⠩⠶ ⠃⠁⠞⠮⠎ ⠁⠗⠁⠇⠙⠁⠗⠮⠝⠙⠁ ⠕⠗⠝⠁⠇⠁⠎⠹⠁⠝⠲
+  - - 1)	
+    - ⠼⠁⠜
+
+# Spaces
```

Comment:
Why are these spaces tests in here? Are they left over from some older yaml file or is the space handling specific to Kazakh?

Reply:
These tables are based on ru-litbrl.ctb, the spaces tests were copyed from ru.yaml, there're no any differences

Reply:
@egli Space handling is unique to every table.